GameOfLifeConsole/Game.cs

```diff
@@ -0,0 +1,116 @@
+﻿using GameOfLifeConsole.Services;
+
+namespace GameOfLifeConsole
+{
+    /// <summary>
+    /// Class of Implementation of GameLogic.
+    /// </summary>
+    public class Game
+    {
+        public GameLogic gameLogic;
+        private FileReadSave _fileReadSave = new FileReadSave();
+        private uint MaxRuns = 50;
+        private int runs = 0;
```

Comment:
Private properties should start with underscore and should continue with lower camelCase.
public property name should be in upper CamelCase.

Reply:
Thank you. Done.